Add store namespaces (#2)
ExtensionEngine/tapster: README.md

  await client.get('foo'); // bar
// ...  
```

### Namespaces

> You can namespace your cache instance to avoid key collisions and allow you to clear only a certain namespace while using the same database.

Reviewer: Consider rephrasing this to make it terser:
```suggestion
Namespacing cache instance enables avoiding key collisions and allows clearing only a certain namespace while using the same database.
```
